src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(missing_debug_implementations)]
```

Comment:
Hardcore.

Reply:
> Is there a way to get this only enabled on debug builds? 

Not that I know of.

> I've read that the code that this generates is a bit expensive,

Expensive...for compilation speed? Runtime? I think it used to in some cases generate a lot of code which would slow down compilation a bit, but I think it's really well optimized nowadays (it's builtin to the compiler).

https://doc.rust-lang.org/nightly/unstable-book/compiler-flags/fmt-debug.html exists (linked from https://github.com/johnthagen/min-sized-rust ) but I think you pretty much have to be writing `#[no_std]` to care is my impression, and we are quite far away from that.

>  but it would be kinda nice to find out if any {:?} slips into production code.

I have never had an issue with that, you have to pretty explicitly type `{:?}`, it doesn't exactly flow off the fingertips :smile: